packages/pipeline-editor/tsconfig.json

```
@@ -1,9 +1,10 @@
 {
   "extends": "../../tsconfig-base",
   "compilerOptions": {
+    "module": "commonjs",
     "outDir": "lib",
     "rootDir": "src",
-    "module": "commonjs",
+    "suppressImplicitAnyIndexErrors": true,
```

why are we suppressing the error checks?